packages/fast-components-react-msft/app/app.tsx

```diff
@@ -47,6 +48,7 @@ enum Theme {
 export interface IAppState extends IColorConfig {
     theme: Theme;
     direction: Direction;
+    density: number;
```

Comment:
please add this to the documentation for this package via the `README.md`.

Reply:
@janechu this is in MSFT only; we aren't documenting the others in the readme from what I see. Should we? This is just a configuration of the dev site with custom slider for density.